app/containers/App.jsx

```diff
@@ -16,18 +14,15 @@ const cx = classNames.bind(styles);
  * A better explanation of react-router is available here:
  * https://github.com/rackt/react-router/blob/latest/docs/Introduction.md
  */
-const App = ({children}) => {
-  return (
-    <div className={cx('app')}>
-      <Navigation />
-      <Message />
-        {children}
-    </div>
-  );
-};
+const App = ({children}) => (
+  <div className={cx('app')}>
```

Comment:
it's one of the most used react libs, I think better to keep it explicit. Can remove `/bind` and cx though, and just use classnames directly

when there are no conditions on a class though you can just include it directly, same as your example or import whole css module

```
import styles form './styles'

...
className={styles.app}
```